utils/material_plots_2D.py

```diff
@@ -36,8 +36,9 @@ def main():
 
         entry_x0, entry_lambda, entry_depth = 0.0, 0.0, 0.0
         for i in range(nMat):
-            if entry.material.at(i) == "Air": continue
-
+            # Ignore materials used in beam pipe as vertex material budget does not work without beam pipe. You can add more materials to ignore here
```

Comment:
This makes this script very non-generic and confusing to use.
Maybe you add an option to ignore a list material names?

Reply:
Yes! Thanks!